src/components/pages/HomePage/HomePage.tsx

```diff
@@ -5,20 +5,15 @@ import PageWrapper from "../PageWrapper/PageWrapper";
 
 const HomePage: React.FC = () => {
   const auth = useAuth();
+  let message = "This app is under construction...";
```

Comment:
Use `const` instead of `let` since the value is not going to change. See: https://medium.com/dailyjs/use-const-and-make-your-javascript-code-better-aac4f3786ca1